app/models/manageiq/providers/cloud_manager.rb

```diff
@@ -28,6 +28,9 @@ class << model_name
     has_many :key_pairs,                     :class_name  => "AuthPrivateKey", :as => :resource, :dependent => :destroy
     has_many :host_aggregates,               :foreign_key => :ems_id, :dependent => :destroy
 
+    virtual_has_many :cloud_networks,  :uses => {:network_manager => :cloud_networks}
+    virtual_has_many :security_groups, :uses => {:network_manager => :security_groups}
```

Comment:
@Fryguy making that change now. I had tried that originally but for some reason I wasn't seeing that work with the API, now I am. weird.